### PR TITLE
New: Media Control - Attach media from URL [ED-1733]

### LINKED ISSUE #413: No option to bring add image by inserting a URL
hello,
I love your plugin but can't use it to create posts because the fastest way to create them is by inserting an image URL. Could you pls add this capability to your plugin.
thanks

### PR DESCRIPTION
**Pull request by StyleShit**
## Before:

image

## After:

image

image

Closes #413 
Closes #13911

### CODE REVIEW

File: assets/dev/js/editor/controls/media.js [hunk 1]
```diff
@@ -110,6 +113,35 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
...
+		elementsToRemove.forEach( ( selector ) => {
+			const el = frameElement.querySelector( selector );
+
+			if ( el ) {
+				el.remove();
+			}
+		} );
+
+		// Remove elements from the URL upload tab.
+		frameElement.classList.add( 'unwanted-elements-removed' );
```
Comment (matipojo) [inline assets/dev/js/editor/controls/media.js:139]:
```suggestion
		frameElement.classList.add( 'elementor-unwanted-elements-removed' );
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/controls/media.js (hunk 1 above)
Comment (StyleShit) [reply assets/dev/js/editor/controls/media.js]: You're right. Changed it to a better name.

**Review by matipojo — DISMISSED** [review]

**Review by matipojo — DISMISSED** [review]

**Review by matipojo — APPROVED** [review]

File: assets/dev/js/editor/controls/media.js [hunk 2]
```diff
@@ -110,6 +116,35 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
...
+			}
+		} );
+
+		// Remove elements from the URL upload tab.
+		frameElement.classList.add( 'elementor-wp-media-elements-removed' );
+
+		// Go to the custom upload tab.
+		frameElement.querySelector( '#menu-item-library' ).click();
+	},
+
```
Comment (kobizz) [inline assets/dev/js/editor/controls/media.js:147]: Use jQuery

**Review by kobizz — CHANGES_REQUESTED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]
Please use jQuery

File: assets/dev/js/editor/controls/media.js [hunk 3]
```diff
@@ -152,31 +152,29 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
...
 
-		elementsToRemove.forEach( ( selector ) => {
-			const el = frameElement.querySelector( selector );
-
-			if ( el ) {
-				el.remove();
-			}
-		} );
+		const $elements = $frame.find( elementsToRemove.join( ',' ) );
+		$elements.remove();
```
Comment (matipojo) [inline assets/dev/js/editor/controls/media.js:156]:
```suggestion
		$frame.find( elementsToRemove.join( ',' ) ).remove();
```

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]
Please pull from 3.3.0

File: assets/dev/js/editor/controls/media.js (hunk 3 above)
Comment (StyleShit) [reply assets/dev/js/editor/controls/media.js]: Done.

**Pull request merged**